en/docs/learn/vscode-plugin-samples/rest-data-service.md

```diff
@@ -0,0 +1,90 @@
+# REST Data Service Sample
+
+This sample demonstrates the capability of the Micro Integrator to perform CRUD operations via a data service using the REST protocol.
+
+This sample contains a Data Service called `RESTDataService` that include data sources and queries required to perform the CRUD operations.
+
+## Setup database to run the sample
+
+1. Create a database and a user with the following details and grant that user access to the database.
+
+    ```bash
+    CREATE DATABASE school_db;
+    USE school_db;
+    CREATE USER 'user'@'localhost' IDENTIFIED BY 'password';
+    GRANT ALL PRIVILEGES ON school_db.* TO 'user'@'localhost';
+    ```
+
+2. Create a table and insert some records.
+
+    ```bash
+    CREATE TABLE `students` (
+    `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    `name` varchar(50) DEFAULT NULL,
+    `school` varchar(50) DEFAULT NULL,
+    `grade` varchar(50) DEFAULT NULL,
+    PRIMARY KEY (`id`)
+    );
+    ```
+
+    ```bash
+    INSERT INTO students (name, school, grade) VALUES
+    ('Tim', 'Summer School', 7),
+    ('Peter', 'Burnley High School', 10);
+    ```
+
+## Deploying the sample
+
+1. Open the sample by clicking on the **REST Data Service** card.
+2. Give a folder location to save the sample.
+3. Open the `RESTDataService` data service artifact and update the RDBMS datasource in it by replacing the `URL`, `Username` and `Password` values with your database credentials.
+4. Download the `mysql-connector-j-8.0.32.jar` and add it to the `<PROJECT_WORKSPACE>/deployment/libs` folder.
```

Comment:
Let's add the download link for the JAR file here.

Reply:
Fixed in 8844ccd775aa7132437584a8e0baee0ad4e5982a